static/app/views/performance/browser/resources/resourceSummaryPage/index.tsx

```diff
@@ -148,7 +148,6 @@ function ResourceSummary() {
             groupId={groupId}
             moduleName={ModuleName.RESOURCE}
             transactionName={transaction as string}
-            additionalFields={[HTTP_RESPONSE_CONTENT_LENGTH]}
```

Comment:
Should we also determine the transaction route prop from the module name within the component to be consistent?

Reply:
That makes sense! I think this whole sample panel could use an overhaul though